faqs/galaxy/datasets_sorting_reference_genome.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sorting Reference Genome
+area: datasets  
+box_type: tip        
+layout: faq        
+contributors: [jennaj, nurzhamalyrys] 
+---
+
+Certain tools expect that reference genomes are sorted in [lexicographical order](https://en.wikipedia.org/wiki/Lexicographic_order). These tools are often downstream of the initial mapping tools, which means that a large investment in a project has already been made (i.e. a long mapping process), before a problem with sorting pops up in conclusion layer tools. No one likes to start over!
+
+How to avoid? Always sort your FASTA reference genome dataset at the beginning of a project. Many sources only provide sorted genomes, but double checking is your own responsibility, and super easy in Galaxy. So easy that there isn't even a shared workflow, just a recipe (but feel free to make your own):
+
+Quick lexicographical sort recipe:
```

Comment:
Is this sentence necessary?

Reply:
> Is this sentence necessary?

@beatrizserrano I changed in this way, so that it would be more clear for the reader. How do you think?

Reply:
I've suggested some changes :)